tests/models/test_electrolyzer_n_inputs_yaml.py

```diff
@@ -0,0 +1,448 @@
+# Copyright (c) 2024, RTE (https://www.rte-france.com)
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+
+import math
+from pathlib import Path
+
+from andromede.simulation import OutputValues, TimeBlock, build_problem
+from andromede.study import (
+    ConstantData,
+    DataBase,
+    Network,
+    Node,
+    PortRef,
+    create_component,
+)
+
+"""
+for every following test we have two electrical productions with an electrolyzer converting to a gaz flow
+we always have:
+    first electric production:
+        - p_max = 70
+        - cost = 10
+    second electric production:
+        - p_max = 80
+        - cost = 20
+    gaz production:
+        - p_max = 30
+        - cost = 15
+    first conversion rate:
+        - alpha = 0.7
+    second conversion rate:
+        - alpha = 0.5
+    for a gaz demand of 100
+"""
+
+
+def test_electrolyzer_n_inputs_1(data_dir: Path, lib, lib_sc):
```

Comment:
Add type annotation in all your tests (`lib: Path`) so that your editor could better help you when developing